hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/config/ServerOptions.java

```diff
@@ -294,7 +294,7 @@ public static synchronized ServerOptions instance() {
                     "arthas.ip",
                     "The IP provided by Arthas, it can be accessible from the outside.",
                     disallowEmpty(),
-                    "0.0.0.0"
+                    "127.0.0.1"
```

Comment:
any context for it? change the IP to localhost

Reply:
the context you can see the introduction about this issue. i think 0.0.0.0 means any ip, and arthas will set default password, sometime the ci failed.  

why prefer localhost not 127.0.0.1? the config item provided by arthas named 'arthas.ip'? im not sure localhost as a domain will work.